matplotcheck/base.py

```diff
@@ -756,7 +757,7 @@ def assert_no_legend_overlap(self, message="Legends overlap eachother"):
 
     """ BASIC PLOT DATA FUNCTIONS """
 
-    def get_xy(self, points_only=False, xtime=False):
+    def get_xy(self, points_only=False):
```

Comment:
I have changed the way that `get_xy()` handles time data. Previously, the docstring stated that if `xtime=True`, then the function would return `datetime.datetime` objects. (Because the data was returned in a dataframe and the way Pandas handles datetime data, it would actualy return `Pandas.timestamp` objects.)

The conversions from matplotlib's datetime datatype to `datetime.datetime` objects is inherently inconsistent, and it made comparisons for datetime data much harder in `assert_xydata()`.

Since `assert_xydata()` was the only function really using the `xtime` flag before, and doesn't use it now, I removed the flag.